wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/environment/server/CheEnvironmentEngineTest.java

```diff
@@ -895,6 +895,43 @@ public void shouldBeAbleToRemoveSnapshot() throws Exception {
         verify(instanceProvider).removeInstanceSnapshot(machineSource);
     }
 
+    @Test
+    public void shouldReplaceServiceNameWithContainerNameAndAliasInLinks() {
```

Comment:
You are testing 2 things here (links with and without alias). You could probably split this test in 2 separate tests.